validate.lic

```diff
@@ -712,6 +712,22 @@ class DRYamlValidator
       warn("#{list} contains the following spell names not found in data/base_spells, #{name_warnings} Check spelling and capitalization.") unless name_warnings.empty?
     end
   end
+  
+  def assert_that_caravan_training_skills_are_valid(settings)
+    return unless settings.caravan_training_skills.empty?
+    valid_caravan_skills = ['Augmentation', 'Utility', 'Warding', 'Attunement', 'Performance', 'Appraisal', 'Mechanical Lore',]
```

Comment:
💡 Can `valid_caravan_skills ` be removed?

Reply:
Yep, I need to do a full review to remove a couple vestigial variables that have been called out and that's definitely one of them